ARMTemplates/201-dtl-create-customimage-from-linux-vm/azuredeploy.json

```diff
@@ -14,6 +14,18 @@
         "description": "Resource ID of an existing VM from which the custom image will be created."
       }
     },
+      "linuxOsState": {
+      "type": "string",
+      "allowedValues": [
+        "NonDeprovisioned",
+        "DeprovisionRequested",
+        "DeprovisionApplied"
+        ],
+      "defaultValue": "NonDeprovisioned",
+      "metadata": {
+        "description": "State of Linux on the machine. It can be one of three values NonDeprovisioned, DeprovisionRequested, and DeprovisionApplied"
```

Comment:
Not really just state. It's how we should treat it and prepare it for being a VM Image.